### PR TITLE
Online friends feature

### PR DESCRIPTION
**Pull request by frankcalise**
Initial go at #20 online friends

### CODE REVIEW

**Comment by ornicar** [comment]
Awesome, thanks for that.

Veloce is enjoying long-deserved vacations, and I'm quite busy at the moment, so I apologize for the delays!

**Comment by frankcalise** [comment]
No worries for the delays - I know Veloce is out, he should enjoy himself. I'm in no rush anyway. I'm happy to start tinkering with this stuff, learning along the way and hopefully making some worthwhile contributions.

File: project/src/js/ui/widget/common.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ widgets.backButton = function(title) {
 widgets.gameButton = function() {
   var myTurns = session.myTurnGames().length;
   return m('button.game_menu_button', {
+    'id': settings.general.theme.board(),
```
Comment (veloce) [inline project/src/js/ui/widget/common.js]: I'd go for `className` instead of `id` here

File: project/src/styl/main.styl [hunk 2]
```diff
@@ -81,6 +81,28 @@ main.out
       width 20px
       height 20px
       background url('../../images/game_icon.svg') no-repeat
```
Comment (veloce) [inline project/src/styl/main.styl]: That's interesting stuff! But it's not related to friends feature. Can you extract this and make a separate pull request for the icon theme?

**Comment by veloce** [comment]
Thanks for the work! Indeed I won't be able to test it before two weeks. But just looking at the code it looks like a good start.

File: project/src/styl/main.styl (hunk 2 above)
Comment (frankcalise) [inline project/src/styl/main.styl]: Definitely, will do - didn't mean for it to get pushed into this request.

**Comment by frankcalise** [comment]
You're welcome. I'll look to make another couple of branches for challenge friend and maybe a profile view of a player.

File: project/src/js/ui/widget/common.js (hunk 1 above)
Comment (ornicar) [inline project/src/js/ui/widget/common.js]: `class` it is
